app/src/main/java/com/ncnf/event/EventDB.java

```diff
@@ -0,0 +1,113 @@
+
+
+    public EventDB() {
+        event1.setUuid(uuid1);
+        event1.addTag(new Tag("\uD83D\uDCF1", "Technology"));
+        event1.addTag(new Tag("\uD83D\uDCF1", "Conference"));
+        event1.addTag(new Tag("\uD83D\uDCF1", "Artificial Intelligence"));
+        events.put(event1.getUuid().toString(), event1);
+
+        event2.setUuid(uuid2);
+        event2.addTag(new Tag("\uD83D\uDCF1", "Opera"));
+        event2.addTag(new Tag("\uD83D\uDCF1", "Singing"));
+        event2.addTag(new Tag("\uD83D\uDCF1", "Art"));
+        events.put(event2.getUuid().toString(), event2);
+
+        event3.setUuid(uuid3);
+        event3.addTag(new Tag("\uD83D\uDCF1", "Concert"));
+        event3.addTag(new Tag("\uD83D\uDCF1", "Rock"));
+        event3.addTag(new Tag("\uD83D\uDCF1", "Singing"));
+        events.put(event3.getUuid().toString(), event3);
+
+        event4.setUuid(uuid4);
+        event4.addTag(new Tag("\uD83D\uDCF1", "Technology"));
+        event4.addTag(new Tag("\uD83D\uDCF1", "Conference"));
+        event4.addTag(new Tag("\uD83D\uDCF1", "BitCoin"));
+        events.put(event4.getUuid().toString(), event4);
+
+        event5.setUuid(uuid5);
+        event5.addTag(new Tag("\uD83D\uDCF1", "Museum"));
+        event5.addTag(new Tag("\uD83D\uDCF1", "Exposition"));
+        event5.addTag(new Tag("\uD83D\uDCF1", "Art"));
+        events.put(event5.getUuid().toString(), event5);
+
+        event6.setUuid(uuid6);
+        event6.addTag(new Tag("\uD83D\uDCF1", "Technology"));
+        event6.addTag(new Tag("\uD83D\uDCF1", "Conference"));
+        event6.addTag(new Tag("\uD83D\uDCF1", "Big Data"));
+        events.put(event6.getUuid().toString(), event6);
+
+        event7.setUuid(uuid7);
+        event7.addTag(new Tag("\uD83D\uDCF1", "Theatre"));
+        event7.addTag(new Tag("\uD83D\uDCF1", "Art"));
+        events.put(event7.getUuid().toString(), event7);
+
+        event8.setUuid(uuid8);
+        event8.addTag(new Tag("\uD83D\uDCF1", "Ballet"));
+        event8.addTag(new Tag("\uD83D\uDCF1", "Dance"));
+        event8.addTag(new Tag("\uD83D\uDCF1", "Art"));
+        events.put(event8.getUuid().toString(), event8);
+
+        event9.setUuid(uuid9);
+        event9.addTag(new Tag("\uD83D\uDCF1", "Concert"));
+        event9.addTag(new Tag("\uD83D\uDCF1", "Pop"));
+        event9.addTag(new Tag("\uD83D\uDCF1", "Singing"));
+        events.put(event9.getUuid().toString(), event9);
+
+        event10.setUuid(uuid10);
+        event10.addTag(new Tag("\uD83D\uDCF1", "Opera"));
+        event10.addTag(new Tag("\uD83D\uDCF1", "Singing"));
+        event10.addTag(new Tag("\uD83D\uDCF1", "Art"));
+        events.put(event10.getUuid().toString(), event10);
+    }
+
+    public Event getEvent(String uuid) {
+        return events.get(uuid);
+    }
+
+    public List<Event> toList() {
+        List<Event> ret = new ArrayList<>();
+        for(Event e : events.values()) {
+            ret.add(e);
+        }
+        return ret;
+    }
+
+ }
```

Comment:
I guess this is temporary (waiting for the db connection). Maybe the class could be moved under MockEvents in the test folder, because this will be useful for mocking the db anyway.

Reply:
This could be done more efficiently in an event generator.

Reply:
This was just for a preview of the UI, but this will be deleted as soon as the feed is connected to the database.